blog/2024-05-13-may-releases.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: "May 2024 Releases"
+slug: "2024-05-13-march-releases"
+authors: [beeankha]
+tags: [announcement, conda, conda-build]
+description: |
+  Conda 24.5.0 and conda-build 24.5.0 have been released! 🎉
+image: img/blog/2024-05-13-may-releases/banner.png
+---
+
+The May 2024 releases included updates to conda and conda-build! 🎉 Both of these have been released to both `main` and `conda-forge`.
+
+## Changes in Conda [24.5.0](https://github.com/conda/conda/releases/tag/24.5.0)
+
+To update conda, run:
+
+```bash
+conda install -n base conda=24.5.0
+```
+
+Amongst other changes, this new version of conda enables MSYS2 packages to utilize upstream installation prefixes, adds support for
+CEP-15 `base_url` field in `repodata.json`, and allows specification of channel URLs using a glob-like wildcard pattern in custom
```

Comment:
```suggestion
[CEP-15](https://github.com/conda/ceps/blob/main/cep-15.md) `base_url` field in `repodata.json`, and allows specification of channel URLs using a glob-like wildcard pattern in custom
```

It would be nice to directly link to the CEP the first time it is mentioned.